pkgs/os-specific/linux/busybox/default.nix

```diff
@@ -31,6 +31,14 @@ let
     CONFIG_FEATURE_UTMP n
     CONFIG_FEATURE_WTMP n
   '';
+
+  debianName = "busybox_1.30.1-5";
+  debianTarball = fetchTarball {
```

Comment:
you should use fetchzip instead of fetchTarball here. fetchTarball is a builtin, and I think has some different behavior on different Nix versions. That might be causing the CI error you're seeing.

Reply:
Done